generators/config/templates/capybara.rb

```diff
@@ -41,6 +41,10 @@ module CapybaraHelpers
 
 Capybara.save_path = Howitzer.log_dir
 
+Capybara::Screenshot.register_driver(Howitzer.driver.to_sym) do |driver, path|
```

Comment:
it is bad approach. Please take a look at https://github.com/mattheworiordan/capybara-screenshot/blob/master/lib/capybara-screenshot.rb#L146

It means we need to add it for drivers where it is really needs and potentially in each case can be own implementation